generators/server/templates/pom.xml.ejs

```diff
@@ -836,6 +836,12 @@
             <groupId>io.micrometer</groupId>
             <artifactId>micrometer-registry-prometheus</artifactId>
         </dependency>
+        <!-- fix https://github.com/reactor/BlockHound/issues/311 -->
+        <dependency>
+            <groupId>io.micrometer</groupId>
+            <artifactId>context-propagation</artifactId>
+            <version>1.0.0</version>
+        </dependency>
```

Comment:
Can you please add another issue so we can track this and remove it when it's fixed?

Reply:
Yes, the issue is #21359